src/transforms/BindAttributeTransform.js

```diff
@@ -66,10 +68,15 @@ module.exports = class BindAttributeTransform {
                 choose either one-way or two-way data binding!`);
             return;
         }
+        // If they already had an onChange handler, make our event handler call theirs as well.
         if (PathUtils.getAttribute(path, eventName)) {
-            PathUtils.warning(path, `Cannot use ${eventName} in conjunction with ${bindAttrName} - you need to choose between
-                either manually handling the event, versus doing it automatically via two-way data binding.`);
-            return;
+            const existingHandler = PathUtils.getAttributeValue(path, eventName).expression;
+            if (t.isArrowFunctionExpression(existingHandler) && t.isAssignmentExpression(existingHandler.body)) {
+                PathUtils.warning(path, `It looks like you're using both ${bindAttrName} and ${eventName} to assign a value. `
+                    + `The 'bind:' attribute automatically adds an ${eventName} handler to save the value; doing both is redundant.`);
+            }
+            PathUtils.deleteAttribute(path, eventName);
+            eventFunction.body = t.sequenceExpression([ eventFunction.body, t.callExpression(existingHandler, eventFunction.params) ]);
```

Comment:
is there a reason to prefer a sequence expression over a block?

Reply:
Just that it was already an arrow function, so I thought it'd be more concise.